src/runtime/manifest-parser.peg

```diff
@@ -301,13 +301,14 @@ ParticleClaimStatement
   / ParticleClaimDerivesFrom
 
 ParticleClaimIsTag
-  = 'claim' whiteSpace handle:lowerIdent whiteSpace 'is' whiteSpace tag:lowerIdent eolWhiteSpace
+  = 'claim' whiteSpace handle:lowerIdent whiteSpace 'is' not:(' not')? whiteSpace tag:lowerIdent eolWhiteSpace
```

Comment:
As annoying as this is, I think you're supposed to use `whiteSpace` instead of hardcoding a single space in your string. We want to support accidental extra spaces in between "is" and "not".

So this should be something like: `not:(whiteSpace 'not')?` I think you can still do `not != null` below.

Actually, doing it the other way around might help your issue with it greedily matching `notTrusted`, e.g. `whiteSpace not:('not' whiteSpace)?`